src/Microsoft.DncEng.SecretManager/Commands/SynchronizeCommand.cs

```diff
@@ -152,14 +157,17 @@ public override async Task RunAsync(CancellationToken cancellationToken)
                 }
             }
 
-            foreach (var (name, key) in manifest.Keys)
+            if (!_verifyOnly)
             {
-                await storage.EnsureKeyAsync(name, key);
-            }
+                foreach (var (name, key) in manifest.Keys)
```

Comment:
Wouldn't this be useful even in verify only mode?

Reply:
It would change things if I ran it in verify mode. The keys are created once, then left alone. I don't believe verifying them is interesting at all.